frontend/src/components/footer.tsx

```diff
@@ -0,0 +1,202 @@
+          setEmail(""); // clear the email input
+        } else {
+          // This is certainly not the best solution to this issue, but it works okay for now.
+          setEmail(""); // clear email input to show error message
+          if (result.error === `400 Bad Request: {"error":"email must be a valid email address"}`) {
+            setErrors(result.error);
+            setPlaceholder("Please enter a valid email address (xxx@domain.com)");
+          } else if (result.error === `400 Bad Request: {"error":"email is already subscribed"}`) {
+            setErrors("Already subscribed!");
+            setPlaceholder("This email is already subscribed!");
+          } else {
+            setErrors(result.error); // show unfilterd error message
+            setPlaceholder("Error with your request, please try again later");
+          }
+        }
+      },
+      (error) => {
+        // If the .then() request fails, show the error message
+        alert(error);
+      },
+    );
+    setLoading(false); // re-enable the submit button
+  };
+
+  return (
+    <footer className={styles.footer}>
+      <div className={styles.container}>
+        {/* NEWSLETTER QUAD */}
+        <div className={styles.newsletterDiv}>
+          <p className={styles.newsletterText}>Join our newsletter to stay updated!</p>
+
+          <form onSubmit={handleNewsletterSubmit}>
+            <div className={styles.subscriptionDiv}>
+              <input
+                type="text"
+                id="email"
+                name="email"
+                placeholder={placeholder}
+                className={errors ? styles.inputBoxError : styles.inputBox}
+                value={email}
+                onChange={handleEmailChange}
+              />
+
+              {submitted ? (
+                <button
+                  type="submit"
+                  id="subscribe-btn"
+                  className={styles.inputButtonSubmitted}
+                  disabled={isLoading}
+                >
+                  Subscribed!
+                  <Image
+                    width={24}
+                    height={24}
+                    className="checkMark"
+                    src="ic_simplesuccess.svg"
+                    alt=""
+                  />
+                </button>
+              ) : (
+                <button
+                  type="submit"
+                  id="subscribe-btn"
+                  className={styles.inputButton}
+                  disabled={isLoading}
+                >
+                  Subscribe
+                </button>
+              )}
+            </div>
+          </form>
+        </div>
+
+        {/* LINK QUAD */}
+        <div className={styles.linkDiv}>
+          <div className={styles.linkRow}>
+            <Link className={`${styles.link} ${styles.boldLink}`} href={links.aboutUs}>
+              About Us
+            </Link>
+            <a className={`${styles.link} ${styles.boldLink}`} href={links.getInvolved}>
```

Comment:
We probably want to be using <Link> tags for all link between pages in the web app

Reply:
Pushed a fix for this, see Slack